docs/topics/tour/kotlin-tour-functions.md

```diff
@@ -148,8 +148,8 @@ fun main() {
 ```
 {kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="kotlin-tour-simple-function-before"}
 
-You can remove the curly braces `{}` and declare the function body using the assignment operator `=`. And due to Kotlin's
-type inference, you can also omit the return type. The `sum()` function then becomes one line:
+You can remove the curly braces `{}` and declare the function body using the assignment operator `=`. When you use the 
+assignment operator `=`, Kotlin uses type inference, so you can also omit the return type. The `sum()` function then becomes one line:
```

Comment:
```suggestion
assignment operator `=`, Kotlin uses type inference, so you can also omit the return type.
(although it is still highly recommended to write it explicitly for public code.)
The `sum()` function then becomes one line:
```

Reply:
I was going to include a link to the [author guidelines](https://kotlinlang.org/docs/api-guidelines-introduction.html) but I then thought that may be too much for an introduction to the language.

Reply:
As discussed, yeah it's a bit heavy a this stage but I will add the tip with slightly different wording!